scripts/bx_auth.sh

```diff
@@ -3,7 +3,7 @@
 BLUEMIX_ORG="Developer Advocacy"
 BLUEMIX_SPACE="dev"
 
-if [[ -z "$BLUEMIX_AUTH" ]]; then
+if [[ -z "$BLUEMIX_API_KEY" ]]; then
```

Comment:
Quick point, this isn't actually testing if we are in a fork, it is only testing if `BLUEMIX_API_KEY` is set. Would it be better to check if [`TRAVIS_PULL_REQUEST`](https://docs.travis-ci.com/user/environment-variables/#Convenience-Variables) is set (see [here](https://github.com/IBM/Kubernetes-container-service-GitLab-sample/blob/master/scripts/bx_auth.sh#L6) for example)?